simulator/src/bin/generate-drawing-examples.rs

```diff
@@ -21,64 +29,64 @@ macro_rules! op {
             $code;
 
             Ok::<(), core::convert::Infallible>(())
-        })().unwrap();
+        })()
+        .unwrap();
+
+        let cleansed_title = $title.replace(" ", "_").to_lowercase();
 
         let output_settings = OutputSettingsBuilder::new().scale(2).build();
 
-        let path = format!("{}/{}.png", OUTPUT_BASE, $title);
+        let file_path = format!("{}/{}.png", IMAGE_OUTPUT_BASE, cleansed_title);
         $display
             .to_image_buffer(&output_settings)
-            .save(&path)
+            .save(&file_path)
             .unwrap();
 
-        let screenshot = base64::encode(std::fs::read(&path).expect("Couldn't open screenshot"));
+        let md_path = format!("./assets/{}.png", cleansed_title);
```

Comment:
This isn't the path that contains the md files.
```suggestion
        let doc_assets_path = format!("./assets/{}.png", cleansed_title);
```